uber/templates/accounts/sitemap.html

```diff
@@ -6,10 +6,12 @@
 
 {% for section, pagelist in pages %}
     <div class="panel panel-default">
-    <div class="panel-heading">{{ section|title }}</div>
+    <div class="panel-heading"><a href="../{{ section }}/index">{{ section|title }}</a></div>
         <ul class="nav nav-pills">
         {% for page in pagelist %}
+            {% if page.name != 'Index' %}
```

Comment:
Unfortunately, because this PR includes commits from another PR (it's really tricky to remember to avoid that), you'll want to rebase this PR off of master in order to get rid of these lines. If you just remove this set of lines in another commit, it'll cause a merge conflict.